src/components/policy/policy_regular/include/policy/policy_manager_impl.h

```diff
@@ -819,6 +822,17 @@ class PolicyManagerImpl : public PolicyManager {
       const std::shared_ptr<policy_table::Table> update,
       const std::shared_ptr<policy_table::Table> snapshot);
 
+  /**
+   * @brief Compares current policies to the updated one.
+   * Trigger actions in case if certain fields after udate was changes.
```

Comment:
`udate` typo, also change `was changes` to `were changed`

Reply:
@jacobkeeler Thanks. Fixed in b07545f011f59e3f00851e5a6db7980a7cd76258